src/main/webapp/components/workspace/workspace.js

```diff
@@ -62,10 +64,18 @@ const workspaceById = gql`
   }
 `
 
-const queryToSearch = query => {
+const queryToSearch = (query, filters) => {
   const { sources, sorts, detail_level, filterTree } = query
+
+  const searchQuery = !filterTree ? defaultQuery : filterTree
```

Comment:
Sometimes, when a search query is swapped with another, and the query wasn't changed from the default anyText *, it's filterTree is null